cluster-autoscaler/main.go

```diff
@@ -183,7 +183,7 @@ var (
 
 	emitPerNodeGroupMetrics = flag.Bool("emit-per-nodegroup-metrics", false, "If true, emit per node group metrics.")
 
-	scaleDownIgnorePDB = flag.Bool("scale-down-ignore-pdb", false, "If true, cluster-autoscaler will ignore PDBs and may evict pods violating PDBs during scale-down.")
+	scaleDownIgnorePDB = flag.Bool("scale-down-ignore-pdb", false, "If true, cluster-autoscaler will ignore PDBs and may evict pods violating PDBs during scale-down by directly deleting pods. This flag does not affect pods controlled by StatefulSets.")
```

Comment:
IMO from the second sentence, it's not super clear which of the following behaviors is taking place:
- CA does not downscale nodes with statefulset pods even if their PDBs can be respected
- CA does not downscale nodes with statefulset pods if any of their PDBs will be violated
- CA does not directly delete statefulset pods when scaling down nodes